tests/utilities/test_storage.py

```diff
@@ -30,3 +34,58 @@ def test_get_flow_image_raises_on_missing_info():
     flow = Flow("test", environment=LocalEnvironment(), storage=Local(),)
     with pytest.raises(ValueError):
         image = get_flow_image(flow=flow)
+
+
+def test_extract_flow_from_file():
+    with tempfile.TemporaryDirectory() as tmpdir:
```

Comment:
Instead of using `tempfile.TemporaryDirectory`, pytest has a `tmpdir` fixture: https://docs.pytest.org/en/stable/tmpdir.html. IMO keeps the tests shorter and more readable.

```
def test_extract_flow_from_file(tmpdir):
    ...
    full_path = tmpdir.join("flow.py")
    ...
```